src/binding/fortran/use_mpi_f08/mpi_c_interface_nobuf.f90

```diff
@@ -493,15 +493,15 @@ function MPIR_Ibarrier_c(comm, request) &
     integer(c_int) :: ierror
 end function MPIR_Ibarrier_c
 
-function MPIR_Op_commutative_c(op, commute) &
+function MPIR_Op_ptr_commutative_c(op, commute) &
```

Comment:
I'm not sure this makes things any clearer, IMO. I would prefer to drop it.

Reply:
I have dropped this commit.